libreoffice-core/sw/source/uibase/app/swmodul1.cxx

```diff
@@ -428,18 +428,6 @@ void SwModule::ClearRedlineAuthors()
     m_pAuthorNames.clear();
 }
 
-static Color lcl_GetAuthorColor(std::size_t nPos)
```

Comment:
I don't think we should remove any unused methods that originate from LOK. This can make future rebasing harder. I think it's best to leave this in